forge-gui/res/editions/Unknown Event Barcelona 2023 Playtest Cards.txt

```diff
@@ -0,0 +1,71 @@
+[metadata]
+Code=UNKBAR23
```

Comment:
try shorten to less than 8
e. g. UEBAR23 etc.

Reply:
Done